.github/workflows/e2e-hotel.yml

```diff
@@ -0,0 +1,151 @@
+name: Hotel-App End-to-End Tests
+
+on:
+  workflow_dispatch:
+#     schedule:
```

Comment:
Do we not want it to run at a scheduled time each week? I have added this to other benchmarks.